Added --suppress_popup flag

lenny-lunarg opened the pull request:
This flag is needed so that if a test fails on windows, the programs can fail instead of hanging.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/KhronosGroup/Vulkan-LoaderAndValidationLayers?pullRequest=625) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/KhronosGroup/Vulkan-LoaderAndValidationLayers?pullRequest=625) before we can accept your contribution.<br/>

davidlunarg commented:
Looks good to me, but I would ask that the commit description be changed to conform to the norm of beginning with the subsystem name, and add a little more detail to the description. I'll suggest:

      demos: Added --suppress_popup flag to cube

lenny-lunarg commented:
Pushed to master

commit 20a497b97273bbbc0f853f12c0cf6db57a2aef9d
Author: lenny-lunarg <lenny@lunarg.com>
Date:   Mon Jun 6 11:07:53 2016 -0600

    demos: Added --suppress_popups flag

    Added a command line option to cube and tri to suppress
    windows message boxes to facilitate Windows testing.

    Change-Id: I069f9b199c9740e1e21c42100ceec42d5fb8f0e2
